### ISSUE TITLE
1.4.3 appdirs broken on anaconda

### ISSUE THREAD

**Issue by Denisevi4** [conda-forge/appdirs-feedstock#10]
Hi,

I don't know where should I ask, but 1.4.3 appdirs broken on anaconda, Can this be fixed?

https://anaconda.org/conda-forge/appdirs/files

**Comment by jakirkham** [comment]
Could you please elaborate on what you mean by broken?

**Comment by Denisevi4** [comment]
Yes, if you look at "Labels" on that webpage that I sent, it says "broken"

**Comment by jakirkham** [comment]
That means there was an issue with the package. So it was pulled from the channel.

**Comment by Denisevi4** [comment]
Also getting this error when conda tries to load it:
ERROR conda.core.link:_execute(502): An error occurred while installing package 'conda-forge::appdirs-1.4.3-py_1'

**Comment by Denisevi4** [comment]
Can this be fixed?

**Comment by jakirkham** [comment]
I'm not sure what you mean. The package doesn't work (as it sound like you are observing). So we have removed it from general consumption and labeled it as broken.

**Comment by Denisevi4** [comment]
It used to work. It became "broken" at most a week ago. Here is an example of our old colda having appdirs of that version from conda-forge. After running conda list:


appdirs                   1.4.3                      py_1    conda-forge

**Comment by Denisevi4** [comment]
we're using python 3.7.1 just to be clear

**Comment by Denisevi4** [comment]
It used to work. It bacame broken probably like a week ago at most:

**Comment by jakirkham** [comment]
Right I’m unclear of the resolution you want. The package had issues so we cannot have it in `main` any more. What else would you like us to do?
